lms/djangoapps/discussion/templates/discussion/discussion_board.html

```diff
@@ -55,16 +55,20 @@
          data-flag-moderator="${flag_moderator}"
          data-user-cohort-id="${user_cohort}">
     <header class="page-header has-secondary">
+        ## Breadcrumb navigation
         <div class="page-header-main">
             <div class="sr-is-focusable" tabindex="-1"></div>
             <div class="has-breadcrumbs"></div>
         </div>
         <div class="page-header-secondary">
+            ## Add Post button
             % if has_permission(user, 'create_thread', course.id):
             <div class="form-actions">
-                <button class="btn btn-small new-post-btn">${_("Add a Post")}</button>
+                <button class="btn-neutral btn-small new-post-btn">${_("Add a Post")}</button>
```

Comment:
+1 to that. I would also move this to btn. 

(Side note: Also - I don't remember the neutral button at all, did we add that recently? )